std_msgs/Header header 

std_msgs/Float64 cable_len
std_msgs/Float64 cable_vel
std_msgs/Float64 cable_acc